samples/rtl/fladd.py

```diff
@@ -0,0 +1,241 @@
+                      src_conn='out',
+                      memlet=dace.Memlet.simple('B_stream', '0'))
+
+# add tasklet memlets
+A = state.add_read('A_stream')
+B = state.add_read('B_stream')
+C = state.add_write('C_stream')
+state.add_memlet_path(A,
+                      rtl_tasklet,
+                      dst_conn='a',
+                      memlet=dace.Memlet.simple('A_stream', '0'))
+state.add_memlet_path(B,
+                      rtl_tasklet,
+                      dst_conn='b',
+                      memlet=dace.Memlet.simple('B_stream', '0'))
+state.add_memlet_path(rtl_tasklet,
+                      C,
+                      src_conn='c',
+                      memlet=dace.Memlet.simple('C_stream', '0'))
+
+# add write_c memlets and access nodes
+write_c_inp = state.add_read('C_stream')
+write_c_out = state.add_write('fpga_C')
+state.add_memlet_path(write_c_inp,
+                      write_c_entry,
+                      write_c,
+                      dst_conn='inp',
+                      memlet=dace.Memlet.simple('C_stream', '0'))
+state.add_memlet_path(write_c,
+                      write_c_exit,
+                      write_c_out,
+                      src_conn='out',
+                      memlet=dace.Memlet.simple('fpga_C', 'i'))
+
+# add copy to device state
+copy_to_device = sdfg.add_state('copy_to_device')
+cpu_a = copy_to_device.add_read('A')
+cpu_b = copy_to_device.add_read('B')
+dev_a = copy_to_device.add_write('fpga_A')
+dev_b = copy_to_device.add_write('fpga_B')
+copy_to_device.add_memlet_path(cpu_a,
+                               dev_a,
+                               memlet=dace.Memlet.simple('A', '0:N//VECLEN'))
+copy_to_device.add_memlet_path(cpu_b,
+                               dev_b,
+                               memlet=dace.Memlet.simple('B', '0:N//VECLEN'))
+sdfg.add_edge(copy_to_device, state, dace.InterstateEdge())
+
+# add copy to host state
+copy_to_host = sdfg.add_state('copy_to_host')
+dev_c = copy_to_host.add_read('fpga_C')
+cpu_c = copy_to_host.add_write('C')
+copy_to_host.add_memlet_path(dev_c,
+                             cpu_c,
+                             memlet=dace.Memlet.simple('C', '0:N//VECLEN'))
+sdfg.add_edge(state, copy_to_host, dace.InterstateEdge())
+
+# validate sdfg
+sdfg.validate()
+
+######################################################################
+
+if __name__ == '__main__':
+
+    # init data structures
+    N.set(8192)
+    a = np.random.randint(0, 100, N.get()).astype(np.float32)
+    b = np.random.randint(0, 100, N.get()).astype(np.float32)
+    c = np.zeros((N.get() // veclen, )).astype(np.float32)
+    print(a.shape, b.shape, c.shape)
+
+    # show initial values
+    print("a={}, b={}".format(a, b))
+
+    # call program
+    sdfg(A=a, B=b, C=c, N=N)
+
+    # show result
+    print("a={}, b={}, c={}".format(a, b, c))
+
+    # check result
+    for i in range(N.get() // veclen):
+        assert c[i] == a[i] + b[i]
+
+    print("Assert passed!")
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/carljohnsen/dace/commit/11984d3c2c997860d2124ef82133077ccd84e753